.github/workflows/e2e-browser.yml

```diff
@@ -14,9 +16,14 @@ jobs:
     runs-on: ubuntu-latest
     environment:
       name: ${{ matrix.environment-name }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         environment-name: ["ESS PodSpaces"]
```

Comment:
```suggestion
        environment-name: ["ESS PodSpaces", "ESS Dev-Next"]
```

this is how I've seen it done elsewhere

Reply:
I'll give it a try, but I don't think this will work, because this will add a `{ env: "Dev-Next", experimental: "false" }` in the run matrix, which we don't want. The `include` block would just add an additional `{ env: "Dev-Next", experimental: "true" }` in any case, since the `experimental: true` flag would override the value in the matrices otherwise, which is always defined and `true`.

Reply:
Yup confirmed, the correct output there is obtained not including `ESS Dev-Next` in the canonical matrix, but only in the `include` block. https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations provides helpful context. 
![correct](https://user-images.githubusercontent.com/8236166/202694321-b9b81881-6b35-4364-94af-c9e1fa54ecf2.png)
![incorrect](https://user-images.githubusercontent.com/8236166/202694324-53acebd7-91b5-4859-b14a-9f873241e549.png)